tests/test_nanobind/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if(Python_FOUND AND TARGET iode)
   #     https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-E-arg-copy_directory
   add_custom_target(test_nanobind
                     COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_BINARY_DIR}/python_api" "."
-                    COMMAND pytest --durations=0
+                    COMMAND pytest --durations=0 -vv
```

Comment:
@jmpplan @yvda23 
the parameters `--durations=0 -vv` passed to the `pytest` command allows to see the list of runned unittests in the console logs + the computation time for each of them